packages/core/src/lib/util.ts

```diff
@@ -34,6 +34,17 @@ const extractZipPromise = promisify(extractZip);
 const DISALLOWED_ANDROID_PACKAGE_CHARS_REGEX = /[^a-zA-Z0-9_\.]/g;
 const VALID_PACKAGE_ID_SEGMENT_REGEX = /^[a-zA-Z][A-Za-z0-9_]*$/;
 
+// List of keywords for Java 11, as listed at
+// https://docs.oracle.com/javase/specs/jls/se11/html/jls-3.html#jls-3.9.
+const JAVA_KEYWORDS = [
```

Comment:
Sorry, but do you mind sorting these?

Reply:
LOL. I actually thought they were sorted, because I copied from the docs... but the docs use column-first sorting and this is using row first... I'll go change that.

Reply:
done